site/content/docs/5.3/content/tables.md

```diff
@@ -243,6 +249,11 @@ Behind the scenes it looks like this:
 
 Add `.table-sm` to make any `.table` more compact by cutting all cell `padding` in half.
 
+<!-- Boosted mod : design callout -->
```

Comment:
You can remove this comment which is not used for other callouts. The shortcode is rather straightforward.